src/overlays/actors/ovl_En_Niw/z_en_niw.h

```diff
@@ -5,11 +5,97 @@
 
 struct EnNiw;
 
+typedef void (*EnNiwActionFunc)(struct EnNiw*, GlobalContext*);
+
+typedef struct {
+    // in OOT this is "type" and there is one instance of an if
+    //  statement specifying `if (true) if (enabled == 1)`
+    //  so in beta there might have been a third value
+    /* 0x0000 */ u8 enabled;
```

Comment:
```suggestion
    /* 0x0000 */ u8 isEnabled;
```
The tendency for bools is to prefix them with `is`